src/components/ButtonGroup/ButtonGroup.stories.tsx

```diff
@@ -73,3 +73,15 @@ export const Disabled = () => {
 		</ButtonGroup>
 	);
 };
+
+/** On Click */
+export const OnClick: Story<IButtonGroupProps> = (args) => {
+	return (
+		<ButtonGroup {...args}>
+			<ButtonGroup.Button onClick={() => alert('zero')}>
```

Comment:
I noticed on ANX react we used console.log() instead of alert().  I prefer alert since it's right in the user's face and they don't have to open the console log, also it doesn't cause linting errors.  Do we have a general consensus on how we want to approach actions like this?